Makefile

```
@@ -45,8 +50,10 @@ docker/unit-tests:
 	@docker run --rm \
 		-e AWS_ACCESS_KEY_ID \
 		-e AWS_SECRET_ACCESS_KEY \
+		-u ${USER_UID}:${USER_GID} \
+		-e HOME=/tmp \
 		-v ${PWD}:${MOUNT_TARGET_DIRECTORY} \
 		${BUILD_TOOLS_DOCKER_IMAGE} \
 		go test -v -timeout 45m -parallel 128 test/terraform_aws_s3_bucket_test.go
```

```suggestion
		go test -v -count=1 -timeout 45m -parallel 128 test/terraform_aws_s3_bucket_test.go
```

`-count =1`  will disable caching of go tests. https://terratest.gruntwork.io/docs/testing-best-practices/avoid-test-caching/